### PR TITLE
WIP: updates to the config to to accept the cookbook name

### PR DESCRIPTION
**Pull request by jdoconnor**
@srhaber

### CODE REVIEW

File: definitions/puma_config.rb [hunk 1]
```diff
@@ -151,17 +151,16 @@
...
-  logrotate_app puma_params[:name] do
-    cookbook "logrotate"
-    path [ puma_params[:stdout_redirect], puma_params[:stderr_redirect] ]
-    frequency "daily"
-    rotate 30
-    size "5M"
-    options ["missingok", "compress", "delaycompress", "notifempty", "dateext"]
-    variables puma_params
-    only_if params[:logrotate]
+  if params[:logrotate]
```
Comment (srhaber) [inline definitions/puma_config.rb:154]: Why is this needed given the `only_if` on line 163?

File: definitions/puma_config.rb [hunk 2]
```diff
@@ -85,7 +85,7 @@
   template "puma_kill.sh" do
     source "puma_kill.sh.erb"
     path "#{params[:puma_directory]}/puma_kill.sh"
-    cookbook "puma"
+    cookbook params[:config_cookbook]
```
Comment (srhaber) [inline definitions/puma_config.rb:88]: Where is this set?

File: definitions/puma_config.rb (hunk 1 above)
Comment (jdoconnor) [inline definitions/puma_config.rb:154]: i can probably remove... the real reason is that I didn't want all the logrotate deps

File: definitions/puma_config.rb (hunk 2 above)
Comment (jdoconnor) [inline definitions/puma_config.rb:88]: https://github.com/lumoslabs/chef-puma/blob/config-updates/definitions/puma_config.rb#L61
